### PR TITLE
Added article about Serpent

### PR DESCRIPTION
**Pull request by mariusc**
So it turns out that I wasn’t able to build the engineering blog locally because I typed `authorId` instead of `authorIds`. It worked now, and I polished and updated the article. The article is post-dated with Monday's date (27 Feb). If we don't manage to publish it that day, please ping me so I can update the date. I really want the date of the article to be the date when we publish it.

Later edit: I updated the date again with Wednesday's date, March 1st. Hope to get it out then :D

### CODE REVIEW

**Comment by mariusc** [comment]
Here's what it looks like on the website, in case you won't run it locally
image

File: source/_posts/2017-03-01-Serpent-more-than-just-another-JSON-mapping-framework.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+<p align=center><img src = "https://d1gwekl0pol55k.cloudfront.net/image/nstack/translate_values/ModelBoiler_5S0fTmSARI.gif"/></p>
+
+
+
+
+### But why build a new one?
+Yes, there are lots of other JSON mapping frameworks out there. But when we started developing Serpent, there was only SwiftyJSON, which wasn't exactly what we wanted. So we needed to build our own. 
+
+Since February 2016 we moved the project to a [public GitHub repo](https://github.com/nodes-ios/Serpent) and kept developing in the open. In the meantime, we were already using this framework in all our apps. At the moment, we can say that ***various versions of Serpent are part of apps with more than 1.3 million monthly users***.
```
Comment (chriscombs) [inline source/_posts/2017-03-01-Serpent-more-than-just-another-JSON-mapping-framework.md]: In February 2016, we moved the project to a [public GitHub repo](https://github.com/nodes-ios/Serpent) and began developing in the open. At the same time, we started using this framework in all our apps, and at the moment, we can say that ***Serpent is used in apps with more than 1.3 million monthly users***.

Comment (chriscombs) [inline source/_posts/2017-03-01-Serpent-more-than-just-another-JSON-mapping-framework.md]: Reworded the first part a bit, and made the bold part more show-off/confident sounding 😄

File: source/_posts/2017-03-01-Serpent-more-than-just-another-JSON-mapping-framework.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+### Working with Serpent
+
+We built Serpent to make the developers' life as easy as possible when creating models. So we also built a few goodies around Serpent, which allow the developer to save a lot of time when creating the models for the app.
```
Comment (chriscombs) [inline source/_posts/2017-03-01-Serpent-more-than-just-another-JSON-mapping-framework.md]: We built Serpent to make developers'

**Comment by mariusc** [comment]
@chriscombs Thanks for the suggestions. I added them 👍

**Pull request merged**